src/scope/component-ops/export-scope-components.ts

```diff
@@ -280,6 +280,9 @@ please run "bit lane track" command to specify a remote-scope for this lane`);
         lane.components.forEach((c) => {
           c.id = c.id.changeScope(remoteName);
         });
+        if (lane.readmeComponent) {
+          lane.readmeComponent.id = lane.readmeComponent.id.changeScope(remoteName);
+        }
```

Comment:
can I export the readme component to one lane, and the lane to another scope?

Reply:
yes you can, you will see in `getUpdatedObjectsToExport`

it checks if it doesnt have a scope and only then changes the scope of the component to the lane component 
```
  lane.components.forEach((c) => {
        c.id = c.id.hasScope() ? c.id : c.id.changeScope(remoteName);
      });
      if (lane.readmeComponent) {
        lane.readmeComponent.id = lane.readmeComponent.id.hasScope()
          ? lane.readmeComponent.id
          : lane.readmeComponent.id.changeScope(remoteName);
      }
      const laneData = { ref: lane.hash(), buffer: await lane.compress() };
      objectList.addIfNotExist([laneData]);
    }
```
I tried to follow how it was handling other components when exporting on a lane, for some reason when exporting to legacy we don't allow exporting to another scope even for lane components

Reply:
I'm not sure how this will work exactly in harmony (or better to say, I don't think we really need this on harmony)
because readmeComponent is a new component, which means its id will always have a scope (either a real scope or a default scope)
@davidfirst am I right here?

Reply:
Actually taking a look a bit down, I see that 
`export type LaneReadmeComponent = { id: BitId; head: Ref | null };`
which means it's an old Id